gpMgmt/bin/README.md

```diff
@@ -12,7 +12,7 @@ To run any of these python scripts, necessary libraries must be installed, and P
 PYTHONPATH="\$GPHOME/lib/python:${PYTHONPATH}"
 ```
 
-This will be set automatically with a `source $GPHOME/greenplum_path.sh`
+This will be set automatically with a `source $GPHOME/Cloudberry_path.sh`
```

Comment:
We don't have the file `Cloudberry_path.sh`, we can still use `greenplum_path.sh` here. Am I right? @avamingli

Reply:
Right.
We can't change things like that. Please recover here @August-beaulo

Reply:
sry it's a mistake and it has been recover